script/Ciel/Day 14/14_01B_03_CIEL14B_4_1B.txt

```diff
@@ -312,30 +329,35 @@ As if to put it on display to the jungles beneath her eyes, the giant spends amp
 [sha:a4000f023826cbdbeeb186bae15f1d3b841e1c00]{
 -- Page 38, Offset 40362.
 -- だが。内臓と共に零れていく苦悶の声には絶望の色はなく、代行者の目には確かな光があった。
-However, rather than despair coming from her anguished voice that spills out together with her organs, the Executor's eyes is filled with light.
+However――even as the Executor's anguished cries are wrung out of her along with her organs,
+ there is no despair in her voice. In fact, there is a definite light in her eyes.
 }
 [sha:bcce2a70d904352abd986f78b667014ba71eb131]{
 -- Page 39, Offset 40363.
 -- 「“なに、その目。不愉快だわ。何か言った？”」
-"What's with those eyes? It makes me sick. Did you say something?"
+"What's with that look?
+ It makes me sick.
+ Did you say something?"
 }
 [sha:137124159320df9d91e6bc134e511a7308a3ce6c]{
 -- Page 39, Offset 40364.
 -- 「……ええ。本当に、どうしようもない、と」
-"...Yeah. I said it really couldn't be helped."
+"Yeah... I said that you really are an idiot."
 }
 [sha:6e564a808063658849a661e184f17f511609e441]{
 -- Page 39, Offset 40365.
 -- 命そのものを指ひとつに支配された状況で、代行者は不敵に笑った。
-In a situation where life itself was ruled by a single finger, the Executor smiles without fear.
+Even though her life lies at the mercy of a single finger,
+ the Executor smiles fearlessly.
 }
 [sha:c60eaa491e9622cc842ea2514c4152449c39ce81]{
 -- Page 40, Offset 40366.
 -- 「まったく―――一度殺されたクセに、まだ彼を解っていないのですね、貴女は」
-"Geez―――Even though you were already killed by him once, you still don't understand him at all."
+"Geez... Even though he already killed you once, you still don't understand him at all, do you?"
```

Comment:
I feel like in Japanese it's a bit more clear that it's like "you don't get what he's doing? Even though he's killed you once already? Dumbass"
>Unbelievable... for someone who has already been killed by him once, you sure don't get what he's up to very well, do you?